Sources/StreamChatUI/UIConfig+ChannelList.swift

```diff
@@ -62,16 +62,7 @@ public extension _UIConfig {
         }
 
         /// The subviews that compose the `SwipeableView`.
-        public struct SwipeableItemSubviews {
-            /// The main content view which you should always use for embedding your cell content.
-            public var cellContentView: UIView.Type = UIView.self
-
-            /// The delete button.
-            public var deleteButton: UIButton.Type = UIButton.self
-
-            /// The more actions button.
-            public var moreActionsButton: UIButton.Type = UIButton.self
-
```

Comment:
... and that's why we also touch every line of just code once during the PR...

Reply:
😅 sorry I thought that I edited this right, dangers of big PRs. :(